test/support/mockConfig.js

```diff
@@ -1,26 +1,143 @@
 const { mock } = require('pactum');
 const { like } = require('pactum-matchers');
-const sheResponse = require('../fixtures/Bundle-DDCC-TX-SHE-response-example-1.json');
+
+const operationOutcomeError = require('../fixtures/OperationOutcome-example.json');
+const rhcResponse = require('../fixtures/Response-ProvideDDCCDocument.json');
 
 const config = {
   setup() {
-    mock.addInteraction({
```

Comment:
Is there a reason the `submitHealthEvent` interaction is removed from `mockConfig.js`? I also saw that `submitHealthEvent.feature` was commented out.

Reply:
I believe I was getting an error when trying to validate against the profile for submitHealthEvent, but your tests pass for submitHealthEvent so we can probably get rid of this commented out code and replace it with the code you've written

Reply:
Looks good now!